packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -46,7 +46,14 @@ const Set<PointerDeviceKind> _kTouchLikeDeviceTypes = <PointerDeviceKind>{
 @immutable
 class ScrollBehavior {
   /// Creates a description of how [Scrollable] widgets should behave.
-  const ScrollBehavior();
+  const ScrollBehavior({ this.updatedAndroidOverscrollIndicator = false});
+
+  /// Indicates which overscroll indicator to use when building the overscroll
+  /// indicator for [TargetPlatform.Android].
+  ///
+  /// Defaults to false, resulting in the [GlowingOverscrollIndicator]. Set to
+  /// true to use the [StretchingOverscrollIndicator]. Cannot be null.
+  final bool updatedAndroidOverscrollIndicator;
```

Comment:
@goderbauer curious for your thoughts, I don't love this opt-in strategy but it was the most straight forward path.

Reply:
I don't love the name. When Android invents a new one, do we then call it the "updatedUpdatedAndroidOverscrollIndicator"? 😝 . Maybe we could call this more descriptively useStretchingOverscrollIndicatorOnAndroid, or something like that?

Also, do we need to wire this through the MaterialScrollBehavior as well?

Reply:
I agree, though, that I am not necessarily in love with this opt-in strategy.

Reply:
How do we expect people to opt-in for the time being? pass A MaterialScrollBehavior with this flag set to MaterialApp.scrollBehavior?

And then at some point we would default that flag to true and then remove it?

Reply:
you could use an enum. I suspect people would want this by default too

Reply:
I've had a think over the weekend and I have another idea for opting in. 

@cedvdb thanks for the input! I think the goal is to give folks time to migrate to the new default first before making the final switch.